external/src/main/java/com/redhat/cloud/policies/engine/lightweight/LightweightEngineImpl.java

```diff
@@ -45,6 +48,11 @@
 @ApplicationScoped
 public class LightweightEngineImpl implements LightweightEngine {
 
+    public static final String USE_ORG_ID = "policies.use-org-id";
```

Comment:
Use the one from NotificationActionPluginListener instead? This way a potential rename would only need to be done once

Reply:
[Bullet-proof approach](https://github.com/RedHatInsights/policies-engine/blob/master/external/src/main/java/com/redhat/cloud/policies/engine/lightweight/LightweightEngineConfig.java): inject the config value in an `@ApplicationScoped` bean and inject that bean everywhere. It even makes the `application.properties` entry optional.

Reply:
The entry should actually be optional already. At least I wanted it to be like that by adding the defaultValue.

Reply:
@gwenneg added. But I left the entry in the application.properties. Maybe I am a bit too shy here

Reply:
I am not into the depth of the framework internals, but I am expecting that they take the value during application start. Either from application.properties or environment variable, then "save" it and inject it everywhere where it's used without checking the status during that process. But that could be wrong.